src/get-started/what-is-exonum.md

```diff
@@ -151,7 +151,8 @@ in enterprise platforms; they have the same principal components.
 
 #### Endpoints
 
-A service has a set of endpoints (realized as REST APIs) using which
+A service has a set of endpoints (realized as REST APIs and WebSockets) using
```

Comment:
A service has a set of REST API endpoints ONLY !!! An access to websockets provided by core.